vision/module/module_orientation.py

```diff
@@ -80,13 +80,21 @@ def get_module_roll(enclosing_region: np.ndarray) -> float:
     #     cv2.drawContours(colorImage, [box], 0, (0, 0, 255), 2)
 
     np_rectangles = np.asarray(rectangles)
-    angles = np_rectangles[:, 2]
 
-    # if the angle is absurd, it won't be taken
-    #   if the module is truly rolled at over 45 degrees, minAreaRect
-    #   should simply choose a different rectangle or axis
-    mask = np.where(abs(angles) < 45)
-    roll = np.mean(angles[mask])
+    try:
```

Comment:
Since we now have failure flags for the pipeline, this try block can be removed form within the algorithm. Unless I'm misunderstanding this and a crash would actually mean 0 roll. But otherwise you can just have 0 be a default value in the try-except block in the pipeline. (along with the flag)

Reply:
The try block is in place so that we can attempt to slice np_rectangles with [:, 2] without a crash if the image is the incorrect shape-- I could set roll to 0 in the pipeline instead, but then it would also set the failure flag, and I'm not sure if that's desirable in this case.